tbd.schema.json

```diff
@@ -123,43 +123,123 @@
           "type": "number"
         },
         "fenestration": {
-          "title": "Window, door, skylight perimeter PSI",
+          "title": "Window or glazed door perimeter PSI",
           "type": "number"
         },
         "head": {
-          "title": "Aperture head/lintel PSI",
+          "title": "Window or glazed door header/lintel PSI",
           "type": "number"
         },
         "headconcave": {
-          "title": "Concave aperture head/lintel PSI",
+          "title": "Concave window or glazed door header/lintel PSI",
           "type": "number"
         },
         "headconvex": {
-          "title": "Convex aperture head/lintel PSI",
+          "title": "Convex window or glazed door header/lintel PSI",
           "type": "number"
         },
         "sill": {
-          "title": "Aperture sill/base PSI",
+          "title": "Window or glazed door sill/threshold PSI",
           "type": "number"
         },
         "sillconcave": {
-          "title": "Concave aperture sill/base PSI",
+          "title": "Concave window or glazed door sill/threshold PSI",
           "type": "number"
         },
         "sillconvex": {
-          "title": "Convex aperture sill/base PSI",
+          "title": "Convex window or glazed door sill/threshold PSI",
           "type": "number"
         },
         "jamb": {
-          "title": "Aperture jamb/side PSI",
+          "title": "Window or glazed door side jamb PSI",
           "type": "number"
         },
         "jambconcave": {
-          "title": "Concave aperture jamb/side PSI",
+          "title": "Concave window or glazed door side jamb PSI",
           "type": "number"
         },
         "jambconvex": {
-          "title": "Convex aperture jamb/side PSI",
+          "title": "Convex window or glazed door side jamb PSI",
+          "type": "number"
+        },
+        "door": {
+          "title": "Opaque door perimeter PSI",
+          "type": "number"
+        },
+        "doorhead": {
+          "title": "Opaque door header/lintel PSI",
+          "type": "number"
+        },
```

Comment:
TBD JSON Hash entries now read in (and write out) **:door** and **:skylight** variants.